src/throttler.guard.ts

```diff
@@ -224,7 +224,11 @@ export class ThrottlerGuard implements CanActivate {
     _throttlerLimitDetail: ThrottlerLimitDetail,
   ): Promise<string> {
     if (!Array.isArray(this.options)) {
-      return this.options.errorMessage || this.errorMessage;
+      if (!this.options.errorMessage) return this.errorMessage;
+
+      return typeof this.options.errorMessage === 'function'
+        ? this.options.errorMessage(_context, _throttlerLimitDetail)
```

Comment:
minor refactor: remove the `_` prefix from the parameters as they're being used now

Reply:
Fair!